### DISCUSSION TITLE
A Record

### DISCUSSION THREAD

**Discussion by Jicknones** [swellstores/community#558] [Help] [answered]
Does anyone know the A Record for Swell? Having hard time with "support," that is literally all I need to update the DNS on GoDaddy for a storefront. They only give you the CNAME.

**Comment by ericingram** [comment]
The CNAME is the record you should create in Godaddy. Here's one of their help articles: https://www.godaddy.com/help/add-a-cname-record-19236

**Reply by Jicknones** [reply]
Yep, already did that--They(GoDaddy) are saying that I need an A record as well. 

Swell support told me this: "For the A records question, this is not possible with our system, as the IP is dynamic and can change."

So now, we are trying a redirect.

**Reply by ericingram** [reply, marked as answer]
The answer from support is correct

You should be able to set the A record to anything but use `www` as the CNAME, then redirect the A record to `www` some other way

I would recommend using a free Cloudflare account for DNS https://dash.cloudflare.com/sign-up. They have some convenient forwarding rules/configurations
